homeassistant/components/vesync/switch.py

```diff
@@ -106,3 +110,57 @@ def __init__(self, switch):
         """Initialize Light Switch device class."""
         super().__init__(switch)
         self.switch = switch
+
+
+class VeSyncHumidifierDisplayHA(VeSyncDevice, SwitchEntity):
+    """Representation of the display on a VeSync humidifier."""
+
+    @property
+    def unique_id(self):
+        """Return the ID of this display."""
+        return f"{super().unique_id}.display"
+
+    @property
+    def name(self):
+        """Return the name of the display."""
+        return f"{super().name} Display"
+
+    @property
+    def is_on(self):
+        """Return True if display is on."""
+        return self.device.details["display"]
+
+    def turn_on(self, **kwargs):
+        """Turn the display on."""
+        self.device.turn_on_display()
+
+    def turn_off(self, **kwargs):
+        """Turn the display off."""
+        self.device.turn_off_display()
+
+
+class VeSyncHumidifierAutomaticStopHA(VeSyncDevice, SwitchEntity):
+    """Representation of the automatic stop toggle on a VeSync humidifier."""
+
+    @property
+    def unique_id(self):
+        """Return the ID of this device."""
+        return f"{super().unique_id}.automatic_stop"
+
+    @property
```

Comment:
Mark entity as `config` (see docs under `entity_category` [here](https://developers.home-assistant.io/docs/core/entity#generic-properties))

```suggestion
    @property
    def entity_category(self):
        """Return the config entity category."""
        return EntityCategory.CONFIG
    
    @property
```

Reply:
Thanks for the suggestion! I'll do that.